Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM pytorch/pytorch:2.0.1-cuda11.7-cudnn8-devel
+
+ENV DEBIAN_FRONTEND=noninteractive \
+    PATH="/opt/miniforge3/bin:$PATH" \
+    TORCH_CUDA_ARCH_LIST="8.0+PTX"
+
+RUN apt update && \
+    apt install -y --no-install-recommends tzdata git unzip wget libglib2.0-0 libgl1-mesa-glx && \
+    wget -O Miniforge3.sh https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh && \
```

Comment:
Is conda really needed? Only `pip install` is used later. Removing conda would reduce the image size.

Reply:
Conda is probably not needed but I have not gotten it to work without conda, despite installing everything using pip. I'd love to not have conda included, so if you can make it work, that would be great!

Reply:
I tried few things to remove conda but it didn't work. I will check your new PR.